README.md

```diff
@@ -1,6 +1,24 @@
 # Baby Project Deployment
-This repository serves as a form of documentation and deployment mechanism 
-through github actions for how we deploy the Baby Project to a server along with
-what security measures we have in place for securing the app and the server(s).
+We use a micro repo structure for a few reasons, but the main one is that we can make use of
+GitHub pages for each repo separately. That way we can deploy docs, test coverage or even a full webapp without extra
+dealing with extra fees. That comes with some drawbacks, but some benefits as well. We publish individual releases from
+each repo and use this repo to deploy them.
 
-<!-- @TODO: add the deployment structure here once we have on -->
+We use [Dokku](https://dokku.com/) for deployment. The main reason is that it's easy to set up and quite easy to scale and 
+maintain. You can however deploy it locally as well using docker.
+
+First build it 🛠
+```shell
+docker build -t baby-project .
```

Comment:
I think it's always better to write the long form of the flags in scripts since it's easier for future readers to grasp what the commands do. E.g. if I don't know/remember what `-t` does, it reads more clearly if it says:
```suggestion
docker build --tag list baby-project .
```

Reply:
What is the `list` there for?

Reply:
I thought it was the long-form of `-t`. Is it only an argument or something? Not sure actually. 🤔
![image](https://user-images.githubusercontent.com/17124245/118764664-8ca7d280-b882-11eb-9b3c-1ae5948f7e69.png)

Reply:
Aaah, it's probably the type of the argument, `string`, `list`, `bytes`, etc. 😄 So should not be included in the long-form version. 👍

Reply:
Yea I removed it in the readme because I saw it worked without it